proposed/extended-coding-style-guide.md

```diff
@@ -247,6 +247,89 @@ class ClassName extends ParentClass implements
 }
 ```
 
+### Using traits
+
+The `use` keyword used inside the classes to implement traits MUST be
+declared one line after the classe's opening brace, and MUST be indented
```

Comment:
I don't like the idea of requiring that a project put the use statement on the line immediately following the opening brace. My own code looks like this:

```php
<?php
namespace Vendor\Package\Feature;

use Vendor\Package\OtherFeature\FeatureFactory;

/**
 * Class docblock
 */
class Feature implements FeatureInterface
{

    use FeatureTrait;

    /**
     * Method docblock
     * @return string
     */
    public function doFeature()
    {
        return 'Done';
    }

}
```

I like to have a single blank line before my use statement and another after my method declarations to give my class brackets a little bit of padding. I don't see any reason we should disallow this case, we should instead say something like "MUST NOT have more than one blank linefeed, MAY omit blank line completely."